Automattic/wp-calypso
# Add oembed provider link for invites section (Take 2)

#### Changes proposed in this Pull Request

* This is another attempt at #60842, which was reverted in #60921  because the changes broke support for invites that had locales in their path.
* In this PR, we again attempt to introduce SSR to the invites section, so we can inject oembed provider links to the document head. The oembed provider links is so we can introduce a preview card feature for shareable invite links, for the P2 onboarding project.
* This PR is different from the one that was reverted in that we add support for routes with locales.

#### Testing instructions
1. As User A, go to https://wordpress.com/me/account and set `Interface language` to anything except English.
2. As User B, send User A an invite to a blog.
3. As User A, open the invite link in the email. You should get the Accept Invite page in your chosen interface language, and not a 404.
4. View Page Source. There should be `<link rel="alternate"  type=application/json+oembed" ...>` tags in the document head.

* Accepting/Declining invites and other invites functionality should be unaffected.
* Opening `<calypso.live container>/accept-invite/<blog-id>/<hash>/fr` should display the page in French. 



Related to 4318-gh-Automattic/p2
